src/components/element-editor/validation/common/validation-child.tsx

```diff
@@ -0,0 +1,38 @@
+import { ChildEntry } from "src/components/common";
+import {
+    ValidationErrorMessage,
+    ValidationRule,
+    ValidationType,
+} from "../common/";
+import { FieldWrapper } from "../validation.styles";
```

Comment:
something to note, the "x" in `tsx` format represents `xml`, we use it when tags are used, e.g. `<FieldWrapper />`, so most styles files can drop the "x" since they wont contain any tags.

Reply:
Got it. Will rename the validation style file in this PR. For the rest of the style files outside of this PR, perhaps I can create another PR in the future to remove all unnecessary "x"s.